infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -187,6 +156,7 @@ public void updateMaxConflictAddress(long newMaxConflictWildcard) {
         log.info("updateMaxConflictAddress, new address: {}", newMaxConflictWildcard);
```

Comment:
You can also remove `updateMaxConflictAddress` once you recreate the cache on reset.